src/main/java/com/ning/http/util/AuthenticatorUtils.java

```diff
@@ -28,8 +27,14 @@
 import com.ning.http.client.spnego.SpnegoEngine;
 import com.ning.http.client.uri.Uri;
 
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+
 public final class AuthenticatorUtils {
     private static final String PROXY_AUTH_HEADER = "Proxy-Authorization";
+    private static final String AVOID_SENDING_NTLM_HEADER = "mule.http.avoidSendingNTLMHeader";
+    private static boolean avoidSendingNtlmHeader = parseBoolean(getProperty(AVOID_SENDING_NTLM_HEADER, "true"));
```

Comment:
Use Boolean.getBoolean which retrieves a system property. 
rename this to alwaysSendNtlmHeader so that the default is false.

Reply:
Done.